portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,18 +27,16 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 
 /** Servlet that returns comments data.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private DatastoreService mDatastore;
+  private DatastoreService mDatastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
nit: if we decide to move it out from #init, then we can consider add 'final' for it